public/components/query_compare/search_result/result_components/result_components.tsx

```diff
@@ -26,11 +26,15 @@ const InitialState = () => {
       grow={true}
       style={{ display: 'flex', justifyContent: 'center', alignItems: 'center' }}
     >
-      <EuiFlexGroup justifyContent="center">
-        <EuiTitle>
-          <h2>Add queries to compare search results.</h2>
-        </EuiTitle>
-      </EuiFlexGroup>
+      <EuiEmptyPrompt
+        iconType="search"
+        title={<h2>No results</h2>}
+        body={
+          <>
+            <p>Add at least one query to display search results.</p>
+          </>
```

Comment:
nit - no need for wrapping element, now that it's just the single `<p>` tag.

```suggestion
            <p>Add at least one query to display search results.</p>
```

Reply:
Also, it's out of scope for this PR, because it's needed everywhere in this plugin, but all user-facing strings should always be wrapped for internationalization. Something along these lines:

```suggestion
          <p>
            <FormattedMessage
                id="searchRelevance.result.noQueryPrompt"
                defaultMessage="Add at least one query to display search results."
              />
            </p>
```

see examples in OpenSearch Dashboards, such as https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/core/public/application/ui/app_not_found_screen.tsx

Reply:
see https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/dev/i18n/README.md

Reply:
Good catch, I can create an issue for this in case someone wants to update all user facing strings